hk2-api/src/main/java/org/glassfish/hk2/utilities/IgnoringErrorService.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2013, 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright should be 2019 Payara Services if this is a new class

Reply:
Sorry, my mistake!